### PR TITLE
[feature/PI-530] Final FHIR Removal

### PR DESCRIPTION
**Pull request by jameslinnell**
swagger refactor

feature/PI-555-remove_fhir_from_create_device

feature/PI-556-remove_fhir_from_read_device

### CODE REVIEW

File: src/api/readDevice/src/v1/steps.py [hunk 1]
```diff
@@ -1,26 +1,57 @@
...
+    return device_repo.read(path_params.device_id)
 
 
-def device_to_fhir_org(data, cache) -> dict:
-    device = data[read_device]
-    fhir_org = create_fhir_model_from_device(device=device)
-    return fhir_org.dict()
+def device_to_dict(data, cache) -> dict:
+    device: Device = data[read_device]
+    return device.state()
```
Comment (jameslinnell) [inline src/api/readDevice/src/v1/steps.py:48]: Needs an HTTP status

Comment (megan-bower4) [reply src/api/readDevice/src/v1/steps.py:48]: I'll add it on read device reference data too as it's also missing there

File: src/layers/domain/response/coding.py [hunk 2]
```diff
@@ -20,10 +17,9 @@ class IssueSeverity(StrEnum):
     INFORMATION = "information"
 
 
-class Coding(_Coding):
-    system: Literal[
-        "https://fhir.nhs.uk/StructureDefinition/NHSDigital-OperationOutcome"
-    ] = CODE_SYSTEM
+class Coding(BaseModel):
+    code = str
+    display = str
```
Comment (jaklinger) [inline src/layers/domain/response/coding.py:22]:
```suggestion
    code: str
    display: str
```

**Review by jaklinger — DISMISSED** [review]

File: infrastructure/swagger/02_info.yaml [hunk 3]
```diff
@@ -87,9 +73,9 @@ info:
 
     | Environment       | Base URL                                                                    |
     | ----------------- | --------------------------------------------------------------------------- |
-    | Sandbox           | `https://sandbox.api.service.nhs.uk/connecting-party-manager/FHIR/R4/`      |
-    | Integration       | `https://int.api.service.nhs.uk/connecting-party-manager/consumer/FHIR/R4/` |
-    | Production        | `https://api.service.nhs.uk/connecting-party-manager/consumer/FHIR/R4/`     |
+    | Sandbox           | `https://sandbox.api.service.nhs.uk/connecting-party-manager/`      |
+    | Integration       | `https://int.api.service.nhs.uk/connecting-party-manager/consumer/` |
+    | Production        | `https://api.service.nhs.uk/connecting-party-manager/consumer/`     |
```
Comment (megan-bower4) [inline infrastructure/swagger/02_info.yaml:78]: Remove /consumer

**Pull request merged**